public/pages/Dashboard/Components/EmptyDashboard/EmptyDashboard.tsx

```diff
@@ -32,7 +32,7 @@ export class EmptyDashboard extends Component<{}, {}> {
             <p>
               Read about{' '}
               <EuiLink
-                href="https://github.com/opendistro-for-elasticsearch/anomaly-detection"
+                href="https://opendistro.github.io/for-elasticsearch-docs/docs/ad/"
```

Comment:
I remember we have another place using wrong link. Can you help fix it? You can search with `href="https://github.com`

Reply:
sure. It seems to be there at `Features.tsx`. Will fix it